test/spec/modules/userId_spec.js

```diff
@@ -148,6 +148,11 @@ describe('User ID', function () {
   });
 
   beforeEach(function () {
+    // TODO: this whole suite needs to be redesigned; it is passing by accident
+    // some tests do not pass if consent data is available
+    // (there are functions here with signature `getId(config, storedId)`, but storedId is actually consentData)
+    // also, this file is ginormous; do we really need to test *all* id systems as one?
+    resetConsentData();
```

Comment:
I agree completely. a lot of this was from the first iteration of the user id module, so a lot of copy/paste from newer modules and signatures weren't necessarily updated as we made changes.

I think we need to have a few tests that do test *all* id systems for basic processing, but the majority of these tests could probably just be mock id systems to test core functionality